JS-Drum-Kit/app.js

```diff
@@ -0,0 +1,32 @@
+const keys = document.querySelectorAll('.key');
+
+// Play drum by keyboard press
+
+function playDrum(e) {
+    const audio = document.querySelector(`audio[data-key="${e.keyCode}"]`);
+    if(!audio) return;
+    audio.currentTime = 0;
+    audio.play();
+    const key = document.querySelector(`.key[data-key="${e.keyCode}"]`);
+    key.classList.add('playing');
+};
+window.addEventListener('keydown' , playDrum);
+
+
+// Remove .playing class after transition action over
+keys.forEach( key => key.addEventListener('transitionend' , removePlayingClass ) );
+function removePlayingClass(e){
+    if( e.propertyName != 'transform' ) return;
+    this.classList.remove('playing');
+};
+
+// Play music by mouse click
+keys.forEach( key => key.addEventListener('click' , playByMouse ));
+function playByMouse(e){
```

Comment:
and this function will be like
```suggestion
function playByMouse(e){
const keyCode = this.getAttribute('data-key');
play(keyCode);
}
```

Reply:
done